### ISSUE TITLE
Create Videos

### ISSUE THREAD

**Issue by markoprljic** [TECommons/coordination#364]

**Comment by mount-manu** [comment]
This _also_ looks appetizing @markoprljic, please assign

**Comment by mount-manu** [comment]
Here's a reference of similar work by the same video producer I'll leverage for this task:
https://user-images.githubusercontent.com/55956086/110774035-fa9ca180-8222-11eb-9435-ff4cacb5acbf.mp4

**Comment by markoprljic** [comment]
Yeah! Looking good @mount-manu I'll keep this epic open and add any video related issues here.

**Comment by mount-manu-zz** [comment]
Closing this; too general: if a particular video idea comes up, I'll open a specific issue for it.
